src/syslog_mcp/tools/analysis/suspicious_analyzer.py

```
@@ -0,0 +1,385 @@
+"""
+Suspicious activity analysis logic.
+
+This module provides pure business logic for analyzing suspicious activity data,
+including pattern detection and risk assessment.
+No data access or presentation logic - just analysis.
+"""
+
+from typing import Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)